Description: fix ftbfs with bison 3.4.1
 For some reason bison exits 1 and fails to generate cfg-grammar.h on
 i386/s390x. Disabling error-verbose, fixes the build failure and is
 the minimal fix to get the project to build. Note that build-system
 appears to be trap return error 1 from bison without failing the
 build when bison fails on lib/cfg-grammar.y. Upstream is working on
 moving to bison 3.4.1 and fixing all deprecated warnings, but that
 patch is not yet merged into master, nor applies cleanly on top of
 3.4.1.
Author: Dimitri John Ledkov <xnox@ubuntu.com>

--- syslog-ng-3.19.1.orig/lib/cfg-grammar.y
+++ syslog-ng-3.19.1/lib/cfg-grammar.y
@@ -79,7 +79,6 @@ extern struct _StatsOptions *last_stats_
 %locations
 %define api.pure
 %pure-parser
-%error-verbose
 
 %code {
 
